tools/docker/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+run() {
+    echo "$*"
+    "$@" || exit $?
+}
+
+scriptdir="$(cd "${0%/*}"; pwd)"
+topdir="${scriptdir%/*/*/*/*}"
+
+main() {
+    run docker container run -u root -v ${topdir}:${topdir} --interactive --tty --rm prplmesh-build $@
```

Comment:
simply use prplmesh-runner instead here, but also base this script on run.sh which added some more capabilities (maybe you need to rebase)